llumnix/llm_engine_manager.py

```diff
@@ -230,24 +234,32 @@ async def _post_migrate(self, rets: List[str], call_migrate_instance_pairs: List
                 if migrate_out_request_ids:
                     migrate_out_request_id = migrate_out_request_ids[0]
                     self.request_instance[migrate_out_request_id] = call_migrate_instance_pairs[i][1]
-                logger.info("{}->{} migrate done, migrate request {}".format(
-                    call_migrate_instance_pairs[i][0], call_migrate_instance_pairs[i][1], migrate_out_request_ids))
-
-    async def _migrate(self) -> None:
-        migrate_instance_pairs = self.global_scheduler.pair_migration()
+                    logger.info("{}->{} migrate done, migrate request {}".format(
+                        call_migrate_instance_pairs[i][0], call_migrate_instance_pairs[i][1], migrate_out_request_ids))
+
+    async def _migrate_control(self) -> None:
```

Comment:
_push_migrations might be better. use nound 'migration' is better than verb 'migrate' is better here. because we do not do migration in this function.